conf/analysers/camunda.analysers.yml

```diff
@@ -0,0 +1,2 @@
+cassandra_keyspace: "benchflow"
```

Comment:
I guess the problem is because you still have the `conf` folder: https://github.com/Cerfoglg/spark-tasks-sender/tree/multiple-dependancies/conf